01_fib/sarahdavy1983/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+// Write to this rather that Stdout directly to help with testing
+var out io.Writer = os.Stdout
+
+// fib returns a slice of integer fibonacci numbers
+func fib(n int) []int64 {
+	switch {
+	case n == 0:
+		return []int64{0, 0}
+	case n == 1:
+		return []int64{0, 1}
+	case n == -1:
+		return []int64{0, 1}
+	case n < -1:
+		x := fib(n + 1)
+		nextVal := x[len(x)-2] - x[len(x)-1]
+		return append(x, nextVal)
+
+	default:
```

Comment:
Sorry, just a nitpick here. Thoughts on using `case n > 1:` here instead of `default:` to make the last case handled clearer to the reader?

Reply:
I understand your concern but after removing the default and replacing with a case the program did not pass the build, so I have reverted back to default case within the switch

Reply:
Thanks for trying that out, I tested that locally and it seems the Go compiler doesn't believe we've handled every case when we use `case n > 1:` and says that not all code paths return a value. 
![image](https://user-images.githubusercontent.com/25007585/59104783-c8874d00-8975-11e9-8ce0-336607b96116.png)


Could theoretically fix it by adding a `return 0` outside of the switch statement but that's wrong on too many levels! Great work :)